src/utils/data_reader.py

```
@@ -0,0 +1,136 @@
+            result = result[result['parameter'].isin(parameters)]
+        
+        return result
+    
+    def read_elevation(self, country: str) -> pd.DataFrame:
+        file_path = self.manager.get_processed_path('terrain') / f"{country.upper()}_elevation_grid.csv"
+        
+        if not file_path.exists():
+            file_path = self.manager.get_processed_path('terrain') / f"{country.lower()}_elevation_grid.csv"
+        
+        if not file_path.exists():
+            raise FileNotFoundError(f"No elevation data found for {country}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)